anaconda.spec.in

```diff
@@ -436,11 +436,11 @@ rm -rf \
 %{_datadir}/cockpit/anaconda-webui/index.js.LEGAL.txt
 %{_datadir}/cockpit/anaconda-webui/index.css.LEGAL.txt
 %{_datadir}/cockpit/anaconda-webui/index.html
-%{_datadir}/cockpit/anaconda-webui/index.js.gz
-%{_datadir}/cockpit/anaconda-webui/index.css.gz
+%{_datadir}/cockpit/anaconda-webui/index.js*
```

Comment:
@jelly now we include the source maps by default for production builds. Do you mind reverting the changes in the anaconda.spec.in to avoid that  we introduce wrong files?


You can enable compression for devel builds as well, that's no problem.

Reply:
The compression is indeed the problem, so that sounds like a good idea.